src/Controller/Component/UsersAuthComponent.php

```diff
@@ -114,6 +118,38 @@ protected function _initAuth()
     }
 
     /**
+     * Check Public Access
+     *
+     * ### Usage
+     *
+     * Your permssions config should return '*' or 'public' for role(s) allowed.
```

Comment:
using role = '*' is ambiguous, we want to identify here actions that do not require users to login, role = '*' means: "logged in users belonging to any role".